modules/module-postgres-storage/README.md

```diff
@@ -31,7 +31,13 @@ storage:
 ```
 
 **IMPORTANT**:
-A separate Postgres server is currently required for replication connections (if using Postgres for replication) and storage. Using the same server might cause unexpected results.
+
+Separate Postgres servers are required for replication connections **if using PostgreSQL versions below 14**.
+
+| PostgreSQL Version | Server configuration                                                                                                                                                                                                     |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Below 14           | Separate servers are required for the source and sync bucket storage. Replication will be blocked if the same server is detected.                                                                                        |
+| 14 and above       | The source database and sync bucket storage database can be on the same server. Using the same database (with separate schemas) is supported but may lead to higher CPU usage. Using separate servers remains an option. |
```

Comment:
Using the same database for the storage will result in many `begin` and `commit` replication slot events being emitted whenever data is written to the tables in the storage schema. These events don't contain any data operations between them, but we do still have to process them. 

Committing these empty transactions would result in a feedback loop where commits generate more transactions etc... The `createEmptyCheckpoints` setting should prevent committing these empty transactions.  This should avoid critical CPU usage, but generally the usage could increase just from using the same database. 

The behaviour of `createEmptyCheckpoints` is configured to only be `false` if using the same Postgres database is used for both the source and storage.